api/serverpb/server.proto

```diff
@@ -138,15 +181,26 @@ message ChangeSettingsRequest {
   google.protobuf.Duration data_retention = 4;
   string ssh_key = 5;
   repeated string aws_partitions = 6;
-  // Prometheus AlertManager URL (e.g., https://username:password@1.2.3.4/path).
+  // External AlertManager URL (e.g., https://username:password@1.2.3.4/path).
   string alert_manager_url = 7;
+  // Remove external AlertManager.
   bool remove_alert_manager_url = 8;
+  // External AlertManager rules.
   string alert_manager_rules = 9;
+  // Remove external AlertManager rules.
   bool remove_alert_manager_rules = 10;
-  // Enable Security Threat Tool
+  // Enable Security Threat Tool.
```

Comment:
Without dot, it will be "title" instead of "description", and is rendered incorrectly in the UI:

![image](https://user-images.githubusercontent.com/11512/100090903-f3fd4080-2e64-11eb-9dde-9353959fc849.png)